api/src/main/java/com/velocitypowered/api/proxy/config/ProxyConfig.java

```diff
@@ -44,8 +43,17 @@
    * Get the MOTD component shown in the tab list.
    *
    * @return the motd component
+   * @deprecated Use {@link #getMotd()} instead
    */
-  Component getMotdComponent();
+  @Deprecated
+  net.kyori.text.Component getMotdComponent();
+
+  /**
+   * Get the MOTD component shown in the tab list.
+   *
+   * @return the motd component
+   */
+  net.kyori.adventure.text.Component getMotd();
```

Comment:
Minor nit: It's a shame this method cannot be named consistently. Open to change in 2.0 once the other gets removed?

Reply:
Yes, as with everything else. (`ProxyConfig` was basically created with the "Extract Interface" refactoring in IntelliJ IDEA, and hasn't really evolved past that. It's a leaky abstraction.)